utils/keycodes.js

```diff
@@ -10,3 +17,36 @@ export const DOWN = 40;
 export const DELETE = 46;
 
 export const F10 = 121;
+
+/**
+ * Check if the access keys and the given character are presssed.
+ *
+ * @param  {KeyboardEvent} event     The event object.
+ * @param  {String}        character The character to check.
+ * @return {Boolean}                 True if the combination is pressed, false if not.
+ */
+export function isAccess( event, character ) {
+	if ( isMac ) {
+		if ( ! event.ctrlKey || ! event.altKey ) {
+			return;
```

Comment:
Though it makes no real difference, strictly speaking this should be `return false`.

Reply:
Yeah, spotted that earlier on too :)